timely_beliefs/beliefs/utils.py

```diff
@@ -493,14 +493,22 @@ def read_csv(
     and the second column has to contain the event values.
     You also need to pass explicit values for the belief horizon/time and cumulative probability,
     in addition to the sensor and source.
-    Useful ddditional kwargs passed to pandas are parse_dates=True and infer_datetime_format=True.
+    Useful additional kwargs passed to pandas are parse_dates=True and infer_datetime_format=True.
```

Comment:
I believe `infer_datetime_format` is only useful in the csv case, and I'd also mention `sheet_name` as something really useful for the excel case:

```
Consult pandas documentation for which additional kwargs can be passed to pandas.read_csv or pandas.read_excel. Useful examples are parse_dates=True, infer_datetime_format=True (for read_csv) and sheet_name (sheet number or name, for read_excel).
```